packages/core/src/actions/smartAccount/sendUserOperation.ts

```diff
@@ -50,5 +50,6 @@ export const sendUserOperation: <
     account,
     uoStruct,
     context,
+    overrides,
```

Comment:
ditto: why do we need overrides passed into the internal _senduserop? Do we use it there? or is this just passed in because overrides are available and there is no reason not to?

Reply:
see above